examples/demo_cisco_tacacs_server.pp

```diff
@@ -0,0 +1,27 @@
+# Manifest to demo cisco_tacacs_server provider
```

Comment:
Thinking about this more and to keep things consistent with other cisco providers I would prefer you not delete/change existing files but create a new file demo_netdev_tacacs_server and add it to a new demo_netdev_all.pp file for providers that could conflict.